service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -99,4 +100,7 @@
     private final List<Element<HearingBooking>> hearingDetails;
     private final List<Element<DocumentBundle>> noticeOfProceedingsBundle;
     private final List<Element<Recipients>> statementOfService;
+    private final C2DocumentBundle temporaryC2Document;
+    @SuppressWarnings("MemberName")
```

Comment:
what is this error and why surpress rather than fix?

Reply:
It's referring to line 105. It does not like memberName starting with a caps - C2DocumentBundle. I'd prefer it reading C2 for consistency.

Reply:
I personally see no problem with c2DocumentBundle. That's java naming convention

Reply:
Ok, lets change it